e2e/wdio/platform/tests/page-footer.e2e-spec.ts

```diff
@@ -0,0 +1,45 @@
+import { PageFooterPo } from '../pages/page-footer.po';
+import {
+    getElementArrayLength,
+    getText,
+    isElementClickable,
+    scrollIntoView,
+} from '../../driver/wdio';
+
+describe('PageFooter test suite:', function() {
+    const pagefooterPage = new PageFooterPo();
+    const {
+        pagefooterclicablelink, pagefootertext  } = pagefooterPage;
+
+    beforeAll(() => {
+        pagefooterPage.open();
+    }, 1);
+    
+        it('verify clickable links ', () => {
+            const linksLength = getElementArrayLength(pagefooterclicablelink);
+            for (let i = 0; i < linksLength; i++) {
+                scrollIntoView(pagefooterclicablelink, i);
+                expect(isElementClickable(pagefooterclicablelink, i)).toBe(true, 'link with index ${i} not clickable');
+            }
+        });
+
+        it('should check pagefooter text', () => {
+            expect(getText(pagefootertext,)).toEqual('1992-2021 @copyright all right reserved.', 'Text is not matching Index');
+            expect(getText(pagefootertext, 1)).toEqual('1992-2021 @copyright all right reserved.', 'Text is not matching Index1');
+            expect(getText(pagefootertext, 2)).toEqual('1992-2021 @copyright all right reserved.', 'Text is not matching Index2');
+        });
+
+    describe('Check visual regression basic', function() {
```

Comment:
please add a skip, as discussed on the previous week's call, we have temporarily skipped visual regression cases until we get a new tool up, due to instability and amount of time QA spends daily on updating screenshots in multiple PRs. to do this, just rename the `describe` to `xdescribe`